arch/src/aarch64/fdt.rs

```diff
@@ -290,9 +290,13 @@ fn create_cpu_nodes(
     fdt.property_u32("#size-cells", 0x0)?;
 
     let num_cpus = vcpu_mpidr.len();
-    let threads_per_core = vcpu_topology.unwrap_or_default().0;
-    let cores_per_package = vcpu_topology.unwrap_or_default().1;
-    let packages = vcpu_topology.unwrap_or_default().2;
+    let (threads_per_core, cores_per_package, packages) = vcpu_topology.unwrap_or((1, 1, 1));
+    if threads_per_core == 0 || cores_per_package == 0 || packages == 0 {
+        error!(
+            "Invalid vcpu topology: threads_per_core: {}, cores_per_package: {}, packages: {}",
+            threads_per_core, cores_per_package, packages
+        );
+    }
```

Comment:
The zeros check is not needed, because it's already done here: https://github.com/cloud-hypervisor/cloud-hypervisor/blob/main/vmm/src/config.rs#L2004C23-L2004C23

If `vcpu_topology` is not none, you should be free from the zero-value risk.

Reply:
Thanks @michael2012z -, fixed.